### PR TITLE
[Shop][Checkout] Seeing discount on order item

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| License         | MIT

### CODE REVIEW

File: features/checkout/seeing_order_summary/seeing_discount_on_order_item_on_summary_page.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@checkout
+Feature: Seeing order item discount
```
Comment (tuka217) [inline features/checkout/seeing_order_summary/seeing_discount_on_order_item_on_summary_page.feature]: **a** order item discount?

File: features/checkout/seeing_order_summary/seeing_discount_on_order_item_on_summary_page.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    In order be aware of discounts applied to an order
```
Comment (tuka217) [inline features/checkout/seeing_order_summary/seeing_discount_on_order_item_on_summary_page.feature]: in order to be(...)

File: features/checkout/seeing_order_summary/seeing_discount_on_order_item_on_summary_page.feature [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        Given the store operates on a single channel in "France"
+        And the store has a product "Lannister Coat" priced at "$100.00"
+        And there is a promotion "Christmas sale"
+        And this promotion gives "10%" off on every product with minimum price at "$50.00"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And I am a logged in customer
+
+    @ui
+    Scenario: Seeing discounted price on order item
```
Comment (tuka217) [inline features/checkout/seeing_order_summary/seeing_discount_on_order_item_on_summary_page.feature]: **a** order item?

File: src/Sylius/Behat/Page/Shop/Checkout/SummaryPage.php [hunk 4]
```diff
@@ -104,4 +149,25 @@ private function getCountryName($countryCode)
...
+     *
+     * @return int
+     */
+    private function getPriceFromString($price)
+    {
+        return (int) round((str_replace(['€', '£', '$'], '', $price) * 100), 2);
+    }
+
+    /**
+     * @param string $total
```
Comment (tuka217) [inline src/Sylius/Behat/Page/Shop/Checkout/SummaryPage.php]: Missing blank line.

File: src/Sylius/Behat/Resources/config/suites/ui/checkout/checkout.yml [hunk 5]
```diff
@@ -20,6 +20,7 @@ default:
                 - sylius.behat.context.setup.payment
                 - sylius.behat.context.setup.product
                 - sylius.behat.context.setup.payment
```
Comment (tuka217) [inline src/Sylius/Behat/Resources/config/suites/ui/checkout/checkout.yml]: Remove this payment.

**Comment by pjedrzejewski** [comment]
Thank you Arek!

**Pull request merged**